models/tiktok_ads.yml

```diff
@@ -91,14 +159,40 @@ models:
     columns:
```

Comment:
are `ad_group_id`s unique across campaigns/advertisers? if not/not sure may be worth adding those ids to this test